app/channel.go

```diff
@@ -158,6 +172,9 @@ func (a *App) CreateChannel(channel *model.Channel, addMember bool) (*model.Chan
 			if cmresult := <-a.Srv.Store.Channel().SaveMember(cm); cmresult.Err != nil {
 				return nil, cmresult.Err
 			}
+			if result := <-a.Srv.Store.ChannelMemberHistory().LogJoinEvent(channel.CreatorId, sc.Id, model.GetMillis()); result.Err != nil {
+				return nil, result.Err
```

Comment:
If this is best effort, I don't think we should return here.

Reply:
(And there's a few other similar places where this comment would also apply).

Reply:
Good call. Fixed in ae2b66b